Header header
float32[360] lidar_values
string[8] additional_data
